CONTRIBUTING.md

```diff
@@ -1,16 +1,281 @@
-# CONTRIBUTE
-This is a short version of the [Development Procedures](https://github.com/RIOT-OS/RIOT/wiki/Development-procedures).
+# Contributing to RIOT
 
-1. Check if your code follows the [coding conventions](https://github.com/RIOT-OS/RIOT/wiki/Coding-conventions). If the code does not comply these style rules, your code will not be merged.
+Thank you for your interest in contributing to RIOT! There are many ways to
+contribute, and we appreciate all of them. You can jump to the major sections
+of this document using the following links:
 
-2. The master branch should always be in a working state. The RIOT maintainers will create release tags based on this branch, whenever a milestone is completed.
+* [Feature Requests][feature-requests]
+* [Bug Reports][bug-reports]
+* [Pull Requests][pull-requests]
+* [Writing Documentation][writing-documentation]
 
-3. Comments on a pull request should be added to the request itself, and *not* to the commit.
+If you have questions, please send an email to users@riot-os.org or
+devel@riot-os.org mailing list or chat on [#riot-os][riot-chat].
 
-4. Keep commits to the point, e.g., don't add whitespace/typo fixes to other code changes. If changes are layered, layer the patches.
+As a reminder, all contributors are expected to follow our
+[Code of Conduct](CODE_OF_CONDUCT.md).
 
-5. Describe the technical detail of the change(s) as specific as possible.
+[riot-chat]: http://webchat.freenode.net?channels=riot-os
 
-6. Use [Labels](https://github.com/RIOT-OS/RIOT/wiki/Labels) to help classify pull requests and issues.
+## Feature Requests
+[feature-requests]: #feature-requests
 
-7. Please adhere to our [code of conduct](https://github.com/RIOT-OS/RIOT/blob/master/CODE_OF_CONDUCT.md)
+Before opening a new feature request, check the
+[existing feature requests][existing-feature-request] if there's one already
+opened on the same topic.
+
+To request new features or enhancements, just open a new
+[feature request issue][new-feature-request]. Describe your use case, why you
+need this feature and why this feature is important for RIOT.
+
+[existing-feature-request]: https://github.com/RIOT-OS/RIOT/issues?q=state:open+type:issue+label:"new+feature"
+[new-feature-request]: https://github.com/RIOT-OS/RIOT/issues/new?template=feature_request.md&title=Feature+Request:+<short+description>
```

Comment:
I'd rather remove the `<short+description>` part. Just from experience I see already a lot of issues titled "{Feature Request|Bug}: <short description>" coming ;-P

Reply:
removed "short description" from title.